src/System.Management.Automation/engine/runtime/Operations/MiscOps.cs

```diff
@@ -403,31 +403,80 @@ private static string GetParameterText(string parameterName)
+                            continue;
+                        // Skip PowerShell magic variables
+                        if (Regex.Match(vName,
+                                "^(global:){0,1}(PID|PSVersionTable|PSEdition|PSHOME|HOST|TRUE|FALSE|NULL)$", 
+                                    RegexOptions.IgnoreCase|RegexOptions.CultureInvariant).Success == false
+                        )
+                        {
+                            updatedScriptblock.Append(scriptblockBodyString.Substring(position, v.Extent.StartOffset - pipelineOffset - position));
+                            updatedScriptblock.Append("${using:");
+                            updatedScriptblock.Append(vName);
+                            updatedScriptblock.Append('}');
+                            position = v.Extent.EndOffset - pipelineOffset;
+                        }
+                    }
+                    updatedScriptblock.Append(scriptblockBodyString.Substring(position));
+                    var sb = ScriptBlock.Create(updatedScriptblock.ToString());
+                    var commandInfo = new CmdletInfo("Start-Job", typeof(StartJobCommand));
+                    commandProcessor = context.CommandDiscovery.LookupCommandProcessor(
+                        commandInfo, CommandOrigin.Internal, false, context.EngineSessionState);
+                    var parameter = CommandParameterInternal.CreateParameterWithArgument(
+                        pipelineAst.Extent, "ScriptBlock", null,
+                        pipelineAst.Extent, sb,
+                        false);
+                    commandProcessor.AddParameter(parameter);
+                    pipelineProcessor.Add(commandProcessor);
+                }
+                else
                 {
-                    var commandsCount = pipelineProcessor.Commands.Count;
-                    if (commandsCount == 1)
+                    for (int i = 0; i < pipeElements.Length; i++)
                     {
-                        // Out-Null is the only command, bail without running anything
-                        return;
+                        commandRedirection = commandRedirections != null ? commandRedirections[i] : null;
+                        commandProcessor = AddCommand(pipelineProcessor, pipeElements[i], pipeElementAsts[i],
+                                                      commandRedirection, context);
                     }
-
-                    // Out-Null is the last command, rewrite command before Out-Null to a null pipe, but
-                    // only if it didn't redirect anything, e.g. `Get-Stuff > o.txt | Out-Null`
-                    var nextToLastCommand = pipelineProcessor.Commands[commandsCount - 2];
-                    if (!nextToLastCommand.CommandRuntime.OutputPipe.IsRedirected)
+    
+                    var cmdletInfo = commandProcessor?.CommandInfo as CmdletInfo;
+                    if (cmdletInfo?.ImplementingType == typeof(OutNullCommand))
```

Comment:
We are now, but you didn't change this code, just indented it, so don't bother changing it now.